torchvision/utils.py

```diff
@@ -282,8 +282,8 @@ def draw_segmentation_masks(
         _log_api_usage_once(draw_segmentation_masks)
     if not isinstance(image, torch.Tensor):
         raise TypeError(f"The image must be a tensor, got {type(image)}")
-    elif image.dtype != torch.uint8:
-        raise ValueError(f"The image dtype must be uint8, got {image.dtype}")
+    elif image.dtype not in {torch.uint8, torch.float32}:
+        raise ValueError(f"The image dtype must be uint8 or float32, got {image.dtype}")
```

Comment:
Here and in the other places, let's just check for `image.is_floating_point()` instead of checking specifically for `float32`. This way we can also support float64, etc.

Reply:
Hey, thanks for replying, this will be fixed with the upcoming commits.